contrib/storage-jdbc/src/test/java/org/apache/drill/exec/store/jdbc/TestJdbcPluginWithMySQLIT.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.jdbc;
+
+import org.apache.drill.PlanTestBase;
+
+import org.joda.time.DateTime;
+
+import org.junit.Test;
+
+
+/**
+ * JDBC storage plugin tests against MySQL.
+ */
+public class TestJdbcPluginWithMySQLIT extends PlanTestBase {
+
+  @Test
```

Comment:
It looks like these classes are very similar. Can we refactor so that only the differences are separate and the rest is a shared based class? I think you can do this by creating an abstract base class that isn't named *Test but has all the test methods. You can then create local implementation of getSchemaName() and what other properties are required that are different to populate the differences.